crates/sui-benchmark/Cargo.toml

```diff
@@ -32,7 +32,7 @@ sui-config = { path = "../sui-config" }
 sui-types = { path = "../sui-types" }
 
 move-core-types = { git = "https://github.com/move-language/move", rev = "7733658048a8bc80e9ba415b8c99aed9234eaa5f", features = ["address20"] }
-narwhal-node = { git = "https://github.com/MystenLabs/narwhal", rev = "57734672c6fd82f761d9e546a7c96c608de3bd85", package = "node" }
+narwhal-node = { git = "https://github.com/punwai/narwhal-1", package = "node" }
```

Comment:
Are we expecting to make updates to Narwhal first and then pin the official crate?

Reply:
Yes, this is until the PR gets merged.